AUTHORS.rst

```diff
@@ -0,0 +1,14 @@
+=======
+Authors
+=======
+
+.. List format (alphabetical order):  Surname, Name. Employer/Affiliation
+
+* Cheeseman, Mark. Vulcan Inc.
```

Comment:
These should include an email for each person

Reply:
hmm I don't have a current email for Mark, and gt4py doesn't include emails:
https://github.com/GridTools/gt4py/blob/master/AUTHORS.rst
Is it ok if I don't?

Reply:
Also, some contributors, like Lucas, might not want their email address listed in a public repo. I doubt he particularly cares, but including emails makes managing this list more cumbersome I think.

Reply:
I like the new no-email style. Git history is a better place to check this sort of thing anyways (it includes emails).

Reply:
If we do not have emails here, then we need to mailmap file. https://blog.developer.atlassian.com/aliasing-authors-in-git/

Reply:
I added it. Now `log` and `shortlog` are cleaned up!